url89849c8c5b_timmolter_XChart_tgz-pJ8-org_knowm_xchart_standalone_Example1J8/src/org/knowm/xchart/standalone/issues/TestForIssue127.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2015-2016 Knowm Inc. (http://knowm.org) and contributors.
+ * Copyright 2011-2015 Xeiam LLC (http://xeiam.com) and contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.knowm.xchart.standalone.issues;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.knowm.xchart.SwingWrapper;
+import org.knowm.xchart.XYChart;
+import org.knowm.xchart.XYChartBuilder;
+
+/**
+ * @author timmolter
+ */
+public class TestForIssue127 {
+
+  public static void main(String[] args) throws InterruptedException, ParseException {
+
+    int[] x = new int[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13 };
+    int[] y = new int[] { 1, 0, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 1 };
+    // int[] x = new int[] { 0, 1, 2, 3, 4, 5, 6, 7 };
+    // int[] y = new int[] { 1, 0, 1, 0, 1, 0, 0, 0 };
+
+    XYChart chart = new XYChartBuilder().width(640).height(480).xAxisTitle("x").yAxisTitle("y").build();
+    chart.setTitle("TEst");
+    chart.getStyler().setLegendVisible(false);
+    new SwingWrapper(chart).displayChart();
+    Thread.sleep(1000);
+
+    chart.addSeries("test", x, y);
+    new SwingWrapper(chart).displayChart();
+    Thread.sleep(1000);
+
+    chart.removeSeries("test");
+    new SwingWrapper(chart).displayChart();
+    Thread.sleep(1000);
+
+    DateFormat sdf = new SimpleDateFormat("dd-HH-mm");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.